appengine/expiration.py

```diff
@@ -0,0 +1,49 @@
+"""
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+"""Delete expired XML.
+"""
+
+__author__ = "fenichel@google.com (Rachel Fenichel)"
+
+
+import storage
+import datetime
+
+from google.cloud import ndb
+
+def delete_expired():
+  # Delete entries that have not been accessed in more than a year.
+  EXPIRATION_DAYS = 365
+  # Limit the query to avoid timeouts.
+  QUERY_LIMIT = 1000
+  bestBefore = datetime.datetime.utcnow() - datetime.timedelta(days=EXPIRATION_DAYS)
```

Comment:
It isn't necessary, but we could also add an expiration time on the entities as a computed property and then you could query for expiry time. 
```
 expiry_time = ndb.ComputedProperty(
      lambda self: self.last_accesssed + datetime.timedelta(days=EXPIRATION_DAYS))
```
Note: ComputedProperty is computed on `put()`, similar to what needed to be done with adding `last_accessed`, if we added the property we would need `put()` the entities before we could query them by that property.

Reply:
Not necessary--it doesn't change what we're doing but is slightly more complicated.